great_expectations/rule_based_profiler/config/__init__.py

```diff
@@ -1,13 +0,0 @@
-from .base import (
```

Comment:
this was good! Why delete?

Reply:
@donaldheppner It made more sense to keep this and other classes under `type`, hence I moved this to `base` just like it is in the `data_context` scheme.  And right now, all imports work, and no circular dependencies.

Reply:
"type" is not a meaningful qualifier -- everything is a Type! I'd really like to preserve the existing, more descriptive, semantic module name we had before, or understand the reason we need to revert.